tensorflow/compiler/xla/literal_util_test.cc

```diff
@@ -966,5 +966,62 @@ TEST_F(LiteralUtilTest, CopyFromProto_Bool) {
   }
 }
 
+// Note that f16 is currently stored in a byte array in little endian byte order
```

Comment:
Perhaps say "serialized" rather than "stored". "stored" is a bit ambiguous because "stored" could mean in memory and in memory the floats are now in the host byte order. Same below.

(I guess we need to add some byte swapping code to ToProto but not in this CL.)